cypress/integration/Case07_Send_money.js

```diff
@@ -81,4 +106,57 @@ describe('Test case 7: Ability to send money', () => {
         })
     })
   })
+
+  it('User is able to send money from exist wallet without "claim"', () => {
+    localStorage.clear()
```

Comment:
were we have large code blocks in the nested Promise callbacks.
could we linearize the code by declaring the test case function async ?

e.g. 
```javascript 
it('User is able to send money from exist wallet without "claim"', async () => {
  // ...
  const sendMoneyUrl = await cy.get('[data-testid*="http"]').invoke('attr', 'data-testid')

  cy.log(sendMoneyUrl)
  // ...
  cy.clearCookies()

  const mnemonic = await cy.readFile('../GoodDAPP/cypress/fixtures/userMnemonicSave.txt')

  StartPage.open()
 // ...
```

Reply:
this block of test is reinsurance if the previous block fails.
in general, to check if the send function works as such

Reply:
@YuryAnanyev the question was - could we use async-await in cypress test cases ? if yes - please rewrite the code how i've shown. It's more readable in this form

Reply:
not used async-await in cypress:
https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Mixing-Async-and-Sync-code
can be viewed in the section "Why can’t I use async / await?"

still open bug about this:
https://github.com/cypress-io/cypress/issues/1417

Reply:
ok

Reply:
it's not good solution because we return file and lost variable `validMoneyLnk`

Reply:
done. without extra nesting levels